common/templates/options.php

```diff
@@ -146,12 +146,19 @@
                 <th scope="row"></th>
                 <td>
                     <label>
-                        GTIN key
+                        GTIN/EAN key
                         <select name="<?= $plugin->getOptionName('custom_gtin'); ?>">
                             <option value=""><?= $plugin->getActiveGtinPlugin() ? __('Automatic detection', 'webwinkelkeur') . ' (' . (explode('/', $plugin->getActiveGtinPlugin())[0] ?? '') . ')' : 'Select key'; ?></option>
-                            <?php foreach ($plugin->getProductKeys() as $key): ?>
-                                <option value="<?= $key['option_value']; ?>" <?= $key['option_value'] == $config['custom_gtin'] ? 'selected' : ''; ?>><?= $key['label']; ?></option>
-                            <?php endforeach; ?>
+                            <optgroup label="<?= _e('Suggested keys', 'webwinkelkeur'); ?>">
+                                <?php foreach ($plugin->getSelectOptions($config['custom_gtin'], true) as $select_option): ?>
+                                    <?= $select_option; ?>
+                                <?php endforeach; ?>
```

Comment:
Why don't you just return a string from `getSelectOptions`?  Would be a lot shorter here.